templates/support/plans-and-pricing.html

```diff
@@ -10,41 +10,43 @@
-      <a class="p-tabs__link js-sticky-tab" href="#openstack">OpenStack</a>
+      <a class="p-tabs__link js-sticky-tab" href="#landscape">Landscape</a>
     </li>
     <li class="p-tabs__item" role="presentation">
-      <a class="p-tabs__link js-sticky-tab" href="#kubernetes">Kubernetes</a>
+      <a class="p-tabs__link js-sticky-tab" href="#security">Security and compliance</a>
     </li>
   </ul>
 </nav>
 
 <section class="p-strip--light is-deep u-no-padding--bottom" id="ua-i-support">
   <div class="row">
-    <div class="col-12">
-      <h2>Ubuntu Advantage for Infrastructure</h2>
-      <p>Ubuntu Advantage for Infrastructure is the professional package of tools, technology and expertise from Canonical, helping organisations around the world get the most out of their Ubuntu deployments. It includes access to: </p>
+    <div class="col-8">
+      <h2>Ubuntu Advantage for Infrastructure (UA-I)</h2>
+      <p>Ubuntu Advantage for Infrastructure is a comprehensive subscription to Canonical technology and expertise, helping organisations around the world get the most out of their Ubuntu deployments. It includes access to:</p>
       <ul class="p-list">
-        <li class="p-list__item is-ticked">Extended Security Maintenance (ESM): critical security updates after Ubuntu End-of-life</li>
-        <li class="p-list__item is-ticked">Landscape: the systems management tool for using Ubuntu at scale</li>
-        <li class="p-list__item is-ticked">Livepatch: automatic kernel security hotfixes without rebooting</li>
-        <li class="p-list__item is-ticked">Knowledge Base: a private archive of expert-written articles and tutorials</li>
-        <li class="p-list__item is-ticked">Support: phone and web-based support at multiple service levels</li>
+        <li class="p-list__item is-ticked">Extended Security Maintenance (ESM) provides critical security updates after an LTS release transitions from the LTS Basic Security Maintenance</li>
+        <li class="p-list__item is-ticked">Landscape system management simplifies use of Ubuntu at scale</li>
+        <li class="p-list__item is-ticked">Livepatch enables automatic kernel security hotfixes without rebooting</li>
+        <li class="p-list__item is-ticked">Knowledge Base offers a private archive of expert-written articles and tutorials</li>
+        <li class="p-list__item is-ticked">Phone and web-based support at multiple service levels</li>
+        <li class="p-list__item is-ticked">FIPS certified modules to meet Common Criteria compliance</li>
+        <li class="p-list__item is-ticked">Ceph and Swift storage, backup and recovery capabilities</li>
       </ul>
-      <p><a href="http://buy.ubuntu.com" class="p-button"><span class="p-link--external">Visit the Ubuntu Advantage store</span></a></p>
+      {% comment %}<p><a href="http://buy.ubuntu.com" class="p-button"><span class="p-link--external">Visit the Ubuntu Advantage store</span></a></p>{% endcomment %}
```

Comment:
Can we not remove this mark up instead of commenting it?

Reply:
we are meant to add it back soon, but if it bothers you I can